repository/Seaside-Core.package/WASessionCookieProtectionFilter.class/instance/privateBrowserSessionID..st

```diff
@@ -0,0 +1,3 @@
+private
+privateBrowserSessionID: aValue
```

Comment:
Traditionally we have not used the `private` prefix for private methods, we simply added them to the private protocol.

Reply:
Agreed; better to keep the code base consistent. I rename them.